### ISSUE TITLE
apstra-cli content import of widgets payload that have been perviously imported with apstra-cli is failing

### ISSUE THREAD

**Issue by mab27** [mab27/Apstra_IBA_Probes#11]
```
apstra-cli> content import --folder /mytmp/IBA_Bootcamp_13_OSPF --type widgets
Importing ['widgets'] resources from /mytmp/IBA_Bootcamp_13_OSPF

Importing widgets resources

Creating new widgets resource "OSPF_Neighbor_RZ_Blue"
Command failed: Unexpected error: Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/aos_cli/command.py", line 538, in run_cmd
    msg = parse_result.cmd.run(parse_result.values, context)
  File "/usr/local/lib/python3.10/site-packages/aos_cli/content_cmds.py", line 305, in run
    self._run(root_path, values, context)
  File "/usr/local/lib/python3.10/site-packages/aos_cli/content_cmds.py", line 363, in _run
    resource = processor.create_resource(specified_res, type_spec)
  File "/usr/local/lib/python3.10/site-packages/aos_cli/content/widget_processor.py", line 128, in create_resource
    res_id=self.aos_session.request('/api/blueprints/%s/iba/widgets' \
  File "/usr/local/lib/python3.10/site-packages/aos/sdk/arest.py", line 236, in request
    response = self._do_http_request(
  File "/usr/local/lib/python3.10/site-packages/aos/sdk/arest.py", line 216, in _do_http_request
    raise RuntimeError(message, res)
RuntimeError: ('expected exit code in [200, 201, 202, 203, 204, 205, 206, 207, 208, 209, 210, 211, 212, 213, 214, 215, 216, 217, 218, 219, 220, 221, 222, 223, 224, 225, 226, 227, 228, 229, 230, 231, 232, 233, 234, 235, 236, 237, 238, 239, 240, 241, 242, 243, 244, 245, 246, 247, 248, 249, 250, 251, 252, 253, 254, 255, 256, 257, 258, 259, 260, 261, 262, 263, 264, 265, 266, 267, 268, 269, 270, 271, 272, 273, 274, 275, 276, 277, 278, 279, 280, 281, 282, 283, 284, 285, 286, 287, 288, 289, 290, 291, 292, 293, 294, 295, 296, 297, 298, 299], got 422. response: {"errors": {"referencing_dashboards": "Unknown field"}}', <Response [422]>)

apstra-cli>
```

**Comment by mab27** [comment]
Filed AOS-44551

**Comment by mab27** [comment]
Fixed, use build 7 of Apstra-cli `4.2.1`
